core/storage/user/gae_models.py

```diff
@@ -108,6 +108,10 @@ class UserSettingsModel(base_models.BaseModel):
     preferred_audio_language_code = ndb.StringProperty(
         default=None, choices=[
             language['id'] for language in constants.SUPPORTED_AUDIO_LANGUAGES])
+    # When the user was created.
+    created_on = ndb.DateTimeProperty(default=None)
+    # When the user settings were last updated.
+    last_updated_on = ndb.DateTimeProperty(default=None)
```

Comment:
Why are you adding these here? These are already defined in the `BaseModel`.

Reply:
@vojtechjelinek Yes, I rechecked and it turns out I won't need some of the changes I'd made including these. We only need to explicitly redefine the `created_on` attribute in the UserSettings model so that the value gets persisted. PTAL, thanks!